fmc/README.md

```diff
@@ -97,41 +99,54 @@ fields may not be changed or removed). Table revisions with different Major Vers
+| fmc_svn_dv_hdl        | 4            | ROM        | Handle of FMC SVN value in the DCCM datavault. |
+| rt_tci_dv_hdl         | 4            | ROM        | Handle of RT TCI value in the DCCM datavault. |
+| rt_cdi_kv_hdl         | 4            | FMC        | Handle of RT CDI value in the Key Vault. |
+| rt_priv_key_ecdsa_kv_hdl    | 4            | FMC        | Handle of RT Alias ECDSA Private Key in the Key Vault. |
+| rt_keygen_seed_mldsa_kv_hdl    | 4            | FMC        | Handle of RT Alias MLDSA Key Generation Seed in the Key Vault. |
+| rt_svn_dv_hdl         | 4            | FMC        | Handle of RT SVN value in the DCCM datavault. |
+| rt_min_svn_dv_hdl     | 4            | FMC        | Handle of Min RT SVN value in the DCCM datavault. |
+| ldevid_tbs_ecdsa_addr       | 4            | ROM        | Local Device ID ECDSA TBS Address. |
+| fmcalias_tbs_ecdsa_addr     | 4            | ROM        | FMC Alias TBS ECDSA Address. |
+| ldevid_tbs_mldsa_addr       | 4            | ROM        | Local Device ID MLDSA TBS Address. |
+| fmcalias_tbs_mldsa_addr     | 4            | ROM        | FMC Alias TBS MLDSA Address. |
+| ldevid_tbs_ecdsa_size       | 2            | ROM        | Local Device ID ECDSA TBS Size. |
+| fmcalias_tbs_ecdsa_size     | 2            | ROM        | FMC Alias TBS ECDSA Size. |
+| ldevid_tbs_mldsa_size       | 2            | ROM        | Local Device ID MLDSA TBS Size. |
+| fmcalias_tbs_mldsa_size     | 2            | ROM        | FMC Alias TBS MLDSA Size. |
+| pcr_log_addr          | 4            | ROM        | PCR Log Address. |
+| pcr_log_index         | 4            | ROM        | Last empty PCR log entry slot index. |
+| meas_log_addr         | 4            | ROM        | Measurement Log Address. |
+| meas_log_index        | 4            | ROM        | Last empty Measurement log entry slot index. |
+| fuse_log_addr         | 4            | ROM        | Fuse Log Address. |
+| rt_dice_pub_key_ecdsa | 96           | FMC        | RT Alias DICE ECDSA Public Key. |
+| rt_dice_pub_key_mldsa_dv_hdl | 4           | FMC        | RT Alias DICE MLDSA Public Key in the DCCM datavault. |
+| rt_dice_sign_ecdsa     | 96           | FMC        | RT Alias DICE ECDSA signature. |
+| rt_dice_sign_mldsa_dv_hdl     | 4           | FMC        | RT Alias DICE MLDSA signature in the DCCM datavault. |
+| ldevid_cert_sig_ecdsa_r_dv_hdl | 4         | ROM        | Handle of LDevId Certificate ECDSA Signature R Component in the DCCM datavault. |
+| ldevid_cert_sig_ecdsa_s_dv_hdl | 4         | ROM        | Handle of LDevId Certificate ECDSA Signature S Component in the DCCM datavault. |
+| ldevid_cert_sig_mldsa_dv_hdl | 4         | ROM        | Handle of LDevId Certificate MLDSA Signature in the DCCM datavault. |
+| idev_dice_pub_key_ecdsa     | 96           | ROM        | Initial Device ID ECDSA Public Key. |
+| idev_dice_pub_key_mldsa_dv_hdl     | 4           | ROM        | Initial Device ID MLDSA Public Key in the DCCM datavault. |
```

Comment:
Question based on pre-existing code: why are IDevID pub keys propagated, but not LDevID.

No need to change this PR based on that.

Reply:
This is for the IDevIdInfo mailbox command in Runtime.